src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2164,4 +2164,10 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="AlwaysOnTop_RoundCorners.Content" xml:space="preserve">
     <value>Enable round corners</value>
   </data>
-</root>
+  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Description" xml:space="preserve">
+    <value>Applies to json and xml</value>
+  </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Header" xml:space="preserve">
+    <value>Try to format the file before rendering the preview</value>
+  </data>
```

Comment:
Hm, the word "try" always feels so insecure to me... Why can't you guarantee? What do you think about:
```suggestion
  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Header" xml:space="preserve">
    <value>Format json and xml files before rendering</value>
  </data>
```

Reply:
I am implementing extensible  formatters that may increment in future, that's why I prefer a checkbox with text and description.
BTW still a WIP: I agree that text needs to be improved!